src/lib/platform/index.ts

```
@@ -2,70 +2,26 @@ import type { PlatformAdapter } from "./adapter";
+  // Dynamic platform detection based on availability of Node.js modules
+  // Try to detect Node.js environment by attempting to access Node.js modules
+  // If they fail, fallback to React Native
+  
   try {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const g: any = globalThis as any;
-    const env = (typeof process !== "undefined" && (process as any).env) ? (process as any).env : undefined;
-    const forced = env && env.MAXIM_PLATFORM;
+    // Check if we can access Node.js modules without actually requiring them in bundling
+    // Use eval to prevent static analysis from including these in the bundle
+    (0, eval)('require')('os');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)